cog/downtime.py

```
@@ -0,0 +1,85 @@
+DATETIME_FORMAT = "%y-%m-%d %H:%M:%S"
+DOWNTIME_PATH = f"{os.getcwd()}/DataBase/downtime.json"
+
+
+@dataclass
+class Downtime:
+    start: datetime
+    # TODO: will Downtime.end be None?
+    end: datetime = field(default_factory=datetime.now)
+    is_restored: bool = False
+
+    def __contains__(self, timestamp: datetime):
```

Ruff (0.8.2)

Missing return type annotation for special method `__contains__`

Add return type annotation: `bool`

(ANN204)